src/pages/Main/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import CarouselContent from './components/CarouselContent'
+import Arrows from './components/Arrows'
+import Dots from './components/Dots'
+import { IMAGE_SLIDER } from './IMAGE_SLIDER'
+import './Carousel.scss'
+
+const Carousel = () => {
+  const lastIndex = IMAGE_SLIDER.length - 1
+  const [activeCarousel, setActiveCarousel] = useState(0)
+  return (
+    <div className="carouselContainer">
+      <CarouselContent
+        activeCarousel={activeCarousel}
+        imageSlider={IMAGE_SLIDER}
+      />
+      <Arrows
+        prev={() =>
+          setActiveCarousel(activeCarousel < 1 ? lastIndex : activeCarousel - 1)
+        }
+        next={() =>
+          setActiveCarousel(
+            activeCarousel === lastIndex ? 0 : activeCarousel + 1
+          )
+        }
+      />
+      <Dots
+        activeCarousel={activeCarousel}
+        imageSlider={IMAGE_SLIDER}
+        onClick={activeCarousel => setActiveCarousel(activeCarousel)}
```

Comment:
onClick이라는 이름은 어떤 요소에 클릭 이벤트가 일어났을 때를 뜻합니다.
그리고 해당 이름은 이미 있는 약속된 언어이기 때문에
onClick이라는 이름 대신 해당 함수의 역할을 props의 Key값으로 설정해주는 것이 좋습니다.

```suggestion
        onClick={setActiveCarousel}
```
그리고 아래 Dots 컴포넌트의 로직을 봤을 때, 이렇게만 넘겨주고 사용해도 동일하게 적용이 됩니다!